source/grid/grid_generator.cc

```diff
@@ -749,6 +749,33 @@ namespace GridGenerator
     tria.set_all_manifold_ids_on_boundary(0);
   }
 
+
+
+  template <int dim>
+  void
+  general_cell (Triangulation<dim> &tria,
+                const std::vector<Point<dim> > &vertices,
+                const bool colorize)
+  {
+    Assert (vertices.size() == std::pow(2., dim),
```

Comment:
Write the rhs as `GeometryInfo<dim>::vertices_per_cell`. That's an integer, and its name already gives away what it is supposed to be.